src/analysis/normed_space/star/basic.lean

```diff
@@ -46,6 +46,8 @@ variables {𝕜 E α : Type*}
 section normed_star_group
 variables [semi_normed_group E] [star_add_monoid E] [normed_star_group E]
 
+@[simp] lemma nnnorm_star (x : E) : ∥star x∥₊ = ∥x∥₊ := subtype.ext norm_star
```

Comment:
Should `x` be implicit here to match `norm_star`?

Reply:
Probably `norm_star` should be changed instead, but I'd prefer to leave that as a follow-up as I've a bunch of things that depend on this PR already, and it's arguably out of scope.